### PR TITLE
Added Chinmay Mehta in members list

### LINKED ISSUE #1469: [Community] Member Profile: Chinmay Mehta
**Current Behaviour**

@chinmaym07 has been a consistent contributor and community member.

**Desired Situation**
Let's recognizing @chinmaym07 as a contributor and community member by creating a profile on https://layer5ng.netlify.app/community/members.

- GitHub: chinmaym07
- Twitter: 
- LinkedIn: chinmay-mehta-009132193
- Link to profile picture:

---

**Contributor Resources**

The layer5.io website, which uses GatsbyJS and GitHub Pages. Site content is found under the <a href="https://github.com/layer5io/layer5/tree/layer5-ng">`layer5-ng` branch</a>. See <a href="https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md">`layer5` contributing instructions</a>.

### PR DESCRIPTION
**Pull request by 96RadhikaJadhav**
**Description**

This PR fixes #1469 .

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by Jashpatel1 — COMMENTED** [review]
@96RadhikaJadhav Can you also add the bio that Chinamy provided in the issue comment.

**Comment by leecalcote** [comment]
Thank you, @96RadhikaJadhav. 👍

yay, @chinmaym07! 🎈

**Comment by 96RadhikaJadhav** [comment]
> @96RadhikaJadhav Can you also add the bio that Chinamy provided in the issue comment.

Done.

File: collections/_members/chinmay-mehta.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: person
+name: Chinmay Mehta
+Bio: I am an undergraduate student at IIIT Bhubaneswar. I am a Web developer & Open Source enthusiast .I love problem solving & always ready to learn and explore something new everyday.
```
Comment (Jashpatel1) [inline collections/_members/chinmay-mehta.md:4]: It should be in lower case 'bio'.

**Comment by leecalcote** [comment]
Good work here, @96RadhikaJadhav. 

@Jashpatel1, do we have an open issue for this same profile on `layer5-ng` branch?

**Comment by Jashpatel1** [comment]
> Good work here, @96RadhikaJadhav.
> 
> @Jashpatel1, do we have an open issue for this same profile on `layer5-ng` branch?

The issue was for layer5-ng branch. @96RadhikaJadhav Can you create a PR for layer5-ng branch as well?

**Comment by Jashpatel1** [comment]
This can be merged for now. Great work here. Thanks @96RadhikaJadhav

**Review by Jashpatel1 — APPROVED** [review]

**Comment by leecalcote** [comment]
Whoo-hoo! 🎈

**Comment by chinmaym07** [comment]
Thanks a lot  @96RadhikaJadhav  @Jashpatel1  @leecalcote sir. 👍🏻  😀

**Comment by leecalcote** [comment]
Congrats, @chinmaym07. This is what happens when you consistently contribute like this. 🙂 

Thank you, @96RadhikaJadhav! We welcome you to join the Layer5 Community. There's more where this comes from. Jump in - http://slack.layer5.io

**Pull request merged**